tests/shell/shell_recorder/tutorial_wrapper/SyntaxCheck.md

```diff
@@ -0,0 +1,39 @@
+Test1
+```sh
+# Mountpoint:/test
+ # testcomment
+	kdb set /test/a a
+# RET:0
+    kdb set /test/b b
+# should yield 'a'
+# STDOUT:a
+kdb get /test/a
+kdb get /test/a
+a
```

Comment:
Way 1: Cannot we reverse it? First command, then `# STDOUT:a`?

Way 2: What happens with multi-line output?